destroy.py

```diff
@@ -0,0 +1,68 @@
+import subprocess
+import sys
+import os
+import re
+
+input('Empty the s3 bucket(s) manually, then press enter to continue.')
+
+app_name = input('Project name: ')
```

Comment:
If we are using the terraform.tfvars to get the supabase info, shouldn't we also get the project name from it also?

Reply:
This raises an important question about how the tool should behave.

I assumed that, for now while we're in the early stages, you run `destroy,py` from the root level of the supastack repo. Meaning if you've been testing and made many projects, you should input which project you're destroying.

Reply:
Once it navigates in there, then it can read the `terraform.tfvars`